src/main/java/com/jc/tm/database/dao/CommentDao.java

```diff
@@ -1,22 +1,24 @@
 package com.jc.tm.database.dao;
 
 import com.jc.tm.database.entity.Comment;
+
+import java.sql.SQLException;
 import java.util.List;
 
 public interface CommentDao extends BaseDao<Comment> {
 
```

Comment:
all methods should be removed, because we already describe this methods into BaseDao interface